### PR TITLE
feat(aait.web.g3): Add blog detail page

### PR DESCRIPTION
**Pull request by yohannes-07**
- Added blog detail page
- Added Error and Loading components

Screenshot from 2023-08-24 22-35-13

### CODE REVIEW

File: aait/web/group-3/starter_project_group3/app/blogs/[id]/page.tsx [hunk 1]
```diff
@@ -0,0 +1,82 @@
+'use client'
```
Comment (BrookW-83) [inline aait/web/group-3/starter_project_group3/app/blogs/[id]/page.tsx:1]: - remove this use client, it unnecessary

**Review by Segnicho — APPROVED** [review]
LGTM

File: aait/web/group-3/starter_project_group3/app/blogs/[id]/page.tsx (hunk 1 above)
Comment (yohannes-07) [reply aait/web/group-3/starter_project_group3/app/blogs/[id]/page.tsx:1]: NextJS 13 and new versions use useParams from next/navigation which works only on client side

**Review by BrookW-83 — APPROVED** [review]

File: aait/web/group-3/starter_project_group3/app/blogs/[id]/page.tsx [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+  }
+
+  if (error){
+      <Error />    
+  }
+
+  if (blog) {
+    return (
+      <div className="flex flex-col items-center mt-16 md:p-0">
+        <div className="bg-white  rounded-lg shadow-md max-w-screen-lg w-full">
```
[the remark this answers is not in the document]
Comment (yohannes-07) [reply aait/web/group-3/starter_project_group3/app/blogs/[id]/page.tsx]: Fixed

File: aait/web/group-3/starter_project_group3/components/commons/Error.tsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+
+const Error = () => {
```
[the remark this answers is not in the document]
Comment (yohannes-07) [reply aait/web/group-3/starter_project_group3/components/commons/Error.tsx]: Fixed

File: aait/web/group-3/starter_project_group3/components/commons/Error.tsx
[the remark this answers is not in the document]
Comment (yohannes-07) [reply aait/web/group-3/starter_project_group3/components/commons/Error.tsx:1]: Fixed.Okay, I will.

**Review by Segnicho — APPROVED** [review]

**Pull request merged**